outcomes/firefox_desktop/spotlight_engagement.toml

```
@@ -1,24 +1,41 @@
-friendly_name = "Spotlight Engagement - not experiment specific"
+### outcome properties
+friendly_name = "Spotlight Engagement - not experiment specific"  # TODO: is the not experiment specific still accurate?
```

Once support for this is implemented, this description can indeed be updated

```suggestion
friendly_name = "Spotlight Engagement"
```